tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_variables.scss

```diff
@@ -1,9 +1,13 @@
-$primary: {{ INDIGO_PRIMARY_COLOR }};
```

Comment:
Was this intentional to no longer use the defined variable in the README?

Reply:
No. I believe it was done mistakenly. Thanks for informing this, I will create a PR for this.